test/test_view_ops.py

```diff
@@ -274,6 +274,36 @@ def test_view_tensor_split(self, device, dtype):
         for a_split_dim1_tensor in a_split_dim1:
             self.assertTrue(self.is_view_of(a, a_split_dim1_tensor))
 
+    @onlyOnCPUAndCUDA
+    @dtypes(*torch.testing.get_all_dtypes())
+    def test_view_tensor_hsplit(self, device, dtype):
+        a = make_tensor((4, 4, 4), device, dtype, low=-9, high=9)
+        a_hsplit = a.hsplit(2)
+        for a_hsplit_tensor in a_hsplit:
+            self.assertTrue(self.is_view_of(a, a_hsplit_tensor))
+        a[2, 2, 2] = 7
+        self.assertEqual(a_hsplit[1][2, 0, 2], a[2, 2, 2])
+
+    @onlyOnCPUAndCUDA
+    @dtypes(*torch.testing.get_all_dtypes())
+    def test_view_tensor_vsplit(self, device, dtype):
+        a = make_tensor((4, 4, 4), device, dtype, low=-9, high=9)
+        a_vsplit = a.vsplit(2)
+        for a_vsplit_tensor in a_vsplit:
+            self.assertTrue(self.is_view_of(a, a_vsplit_tensor))
+        a[2, 2, 2] = 7
+        self.assertEqual(a_hsplit[1][0, 2, 2], a[2, 2, 2])
```

Comment:
take care of typo in 295,305 for a_hsplit -> a_vsplit and a_hsplit->d_dsplit..  some side note, single tensors generally noted by t (mutiple tensors go by a,b,c,d...)